crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -3001,12 +3009,12 @@ impl AuthorityPerEpochStore {
             estimates,
         } in execution_time_observations
         {
+            let Some(estimator) = execution_time_estimator.as_mut() else {
+                error!("dropping ExecutionTimeObservation from possibly-Byzantine authority {authority:?} sent when ExecutionTimeEstimate mode is not enabled");
```

Comment:
maybe make this a debug_fatal! and then it'll get reported in metrics?

Reply:
the reason I didn't do this is it's not an invariant violation: any (non-)byzantine validator can send us an unexpected message if they feel like it and we should drop it, doesn't mean something broke locally